.github/workflows/proto.yaml

```diff
@@ -0,0 +1,20 @@
+name: Protobuf
+
+on:
+  pull_request:
+    paths:
+      - "**.proto"
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@master
+      - name: lint
+        run: make proto-lint
+  breakage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

Comment:
`actions/checkout@v2` plz

a rude way to fetch master will be in use of [this approach](https://github.com/actions/checkout#Fetch-all-history-for-all-tags-and-branches)
another option to check would be `- run: git fetch --prune --unshallow`. [example](https://github.com/ovrclk/akash/blob/ac8ca601b9d87dff1507e103a6a028ed08064906/.github/workflows/release.yaml#L18)

this should fix [this](https://github.com/ovrclk/akash/pull/803/checks?check_run_id=1224481285)

Reply:
@troian , This approach is also throwing error. Working only with absolute git https url.